waku/v2/utils/peer.go

```diff
@@ -10,6 +12,10 @@ type DialError struct {
 	PeerID peer.ID
 }
 
+// ErrNoPeersAvailable is emitted when no suitable peers are found for
+// some protocol
+var ErrNoPeersAvailable = errors.New("no suitable peers found")
+
```

Comment:
I'm confused on why `ErrNoPeersAvailable` was moved inside `utils`. Isn't this error emitted/used only by the peer manager package?

Reply:
i had to use this in `api/filter.go` line `191` , hence had to move it out.